loadbalanced-fargate-svc/service/infrastructure/cloudformation.yaml

```diff
@@ -93,7 +93,10 @@ Resources:
       HealthCheckTimeoutSeconds: 5
       HealthyThresholdCount: 2
       TargetType: ip
-      Name: '{{service_instance.unique_name}}'
+      # Note that the Name property has a 32 character limit, which could be
+      # reached by using either {{service_name}}, {{service_instance_name}}
+      # or a combination of both as we're doing here
+      Name: '{{service_name}}--{{service_instance_name}}'
```

Comment:
Related: https://github.com/aws-samples/aws-proton-sample-templates/pull/14

Nit: Can this use 1 dash, and not 2?

32 characters seems extremely easy to hit though, and would be a sad experience to have your first run service fail because the combination of service_name and service_instance_name was longer than 32 chars.  Should we remove Name entirely? It's not a required field

Reply:
I like the idea of removing the `Name` field here, I actually thought it was required, my bad.

That being said, I do think it's kinda nice to have a named target group, to prevent confusion in the console for instance. I initially went for the double dash due to the character limitation of that field (my instinct was a slash or underscore), and I thought there was value in having examples using the combinations of `{{service_name}}` and `{{service_instance_name}}`. With the default values provided in this example, I wasn't a fan of the single dash (`front-end-frontend-dev`), as it didn't show the delimitation between the two fields.

With that said, I 100% agree with you, and I think that my preference here would be to only use `{{service_instance_name}}` (`frontend-dev` with the default value here), since it's still leave 

@kohidave Any thoughts? I'm not attached to any approaches and would happily update to the option we think is best.